app/views/layouts/base.html.erb

```diff
@@ -20,7 +20,11 @@
   <%= yield(:meta_tags) if content_for?(:meta_tags) %>
 
   <title>
-    <%= APP_NAME %><%= raw(" - #{yield(:title)}") if content_for?(:title) %>
+    <% if content_for?(:title) %>
+      <%= raw("#{yield(:title)} - #{APP_NAME}") %>
+    <% else %>
+      <%= APP_NAME %>
+    <% end %>
   </title>
```

Comment:
1. how about we add a spec for this new behavior in `spec/views/layouts`?
2. I think we can use rails tag helpers do we don't have to manually escape this ourselves?

```erb
<%= content_tag(
      'title',
      content_for?(:title) ? "#{yield(:title)} - #{APP_NAME}" : APP_NAME,
) %>
```